plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```
@@ -576,17 +573,28 @@ private function get_paypal_order_items( $order ) {
+	/**
+	 * Get the amount for a specific order item.
+	 *
+	 * @param WC_Order      $order Order object.
+	 * @param WC_Order_Item $item Order item.
+	 * @return float
+	 */
+	private function get_paypal_order_item_amount( $order, $item ) {
+		return (float) (
+			'fee' === $item->get_type()
+				? $item->get_amount()
+				: $order->get_item_subtotal( $item, $include_tax = false, $rounding_enabled = false )
```

PHPMD (2.15.0)

Avoid unused local variables such as '$include_tax'. (undefined)

(UnusedLocalVariable)